setup/kubernetes/rke.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Rancher Kubernetes Engine
+description: Learn how to setup a Rancher cluster for your Coder deployment
+---
+
+This deployment guide shows you how to set up a bare-metal Rancher Kubernetes
+Engine (RKE) cluster on which Coder can deploy.
+
+## Prerequisites
+
+You must have at least one Linux host (node) with the following utilities
+installed:
+
+### Binaries
+
+- [Docker](https://docs.docker.com/engine/install/)
+- [Helm](https://helm.sh/docs/intro/install/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [rke](https://rancher.com/docs/rke/latest/en/installation/)
+
+### Storage
+
+Since Coder requires dynamic storage provisioning, you'll need to
+install a [Rancher-supported storage provisioner](https://rancher.com/docs/rancher/v2.5/en/cluster-admin/volumes-and-storage/provisioning-new-storage/#prerequisites).
+We recommend using [Longhorn](https://longhorn.io/), since it is tightly
+integrated with Rancher.
+
+### Networking
+
+Since the Coder default service is of `type: LoadBalancer`, we recommend
+[installing MetalLB into your Rancher cluster](https://metallb.universe.tf/installation/).
+You must install MetalLB prior to installing Coder for MetalLB to identify the
+Coder service and [expose it externally](https://metallb.universe.tf/usage/).
```

Comment:
I recommend linking to Longhorn's recommended CNI providers: 

https://rancher.com/docs/rancher/v2.6/en/faq/networking/cni-providers/